src/vita-make-fself.c

```diff
@@ -4,7 +4,11 @@
 #include <inttypes.h>
 #include <zlib.h>
 
+typedef struct {} vita_export_t;
```

Comment:
I guess not. When I was testing, I didn't have YAML headers. I'll push an update.